frontend/scripts/report-coveralls.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
-
-set -e
-
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
-# Prow env reference: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md
-export COVERALLS_REPO_TOKEN=$(node $DIR/get-coveralls-repo-token.js)
-REPO_BASE="https://github.com/kubeflow/pipelines"
-export CI_PULL_REQUEST="$REPO_BASE/pull/$PULL_NUMBER"
-
-export COVERALLS_SERVICE_JOB_ID="${PROW_JOB_ID}-frontend"
-export COVERALLS_SERVICE_NAME="prow-frontend"
-cat ./coverage/lcov.info | npx coveralls
-
-export COVERALLS_SERVICE_JOB_ID="${PROW_JOB_ID}-frontend-server"
-export COVERALLS_SERVICE_NAME="prow-frontend-server"
-cat ./server/coverage/lcov.info | npx coveralls
```

Comment:
@DharmitD can you provide a description for what is being removed, and why it's no longer needed? this looks like it's utilizing[ npm coveralls](https://www.npmjs.com/package/coveralls), but I don't see this being utilized in the new GHA that were introduced. Are we losing testing coverage reporting in this migration to GHA?

Reply:
Yes, this is coverage reporting for front end tests. We took this out since these are based in Prow.

@jlyaoyuli @zijianjoy @chensun could you confirm whether we need these, or it would be okay to take these out?